### PR TITLE
Make checking nvidia site interval configurable

### PR DESCRIPTION
**Pull request by Senaxx**
Make checking Nvidia site interval configurable. The default setting is now 3 seconds but can be easily changed to anything the user likes.

### CODE REVIEW

File: stores/nvidia.py [hunk 1]
```diff
@@ -222,6 +222,7 @@ def __init__(self, gpu, locale="en_us"):
                     self.nvidia_password = self.config["NVIDIA_PASSWORD"]
                     self.auto_buy_enabled = self.config["FULL_AUTOBUY"]
                     self.cvv = self.config.get("CVV")
+                    self.interval=int(self.config.get("INTERVAL"))
```
Comment (Hari-Nagarajan) [inline stores/nvidia.py:225]: This will cause errors for existing users until they update the config

Comment (Senaxx) [reply stores/nvidia.py:225]: Oh yeah, I see what you mean, didn't think of that.  I'll try to create a default value of 5, and it's getting overridden with this value if it's get's added.

Comment (aljcalandra) [reply stores/nvidia.py:225]: Change it to `self.interval=int(self.config.get("INTERVAL", 3))`

**Comment by Senaxx** [comment]
Deleted, I'll push a new PR with comments from @Hari-Nagarajan  and @aljcalandra

**Pull request closed without merging**